client/src/app/site/common/components/start/start.component.html

```diff
@@ -8,6 +8,6 @@ <h2 translate>Home</h2>
     <div class="app-content" translate>
         <h1>{{ welcomeTitle | translate }}</h1>
 
-        <div [innerHTML]="welcomeText | translate"></div>
```

Comment:
This still needs to be translated, but this can be done then getting the config value, before sanitizing it.